app/src/ui/add-repository/add-existing-repository.tsx

```diff
@@ -68,9 +68,19 @@ export class AddExistingRepository extends React.Component<
   }
 
   public async componentDidMount() {
-    const isRepository = await isGitRepository(this.state.path)
+    const pathToCheck = this.state.path
+    if (pathToCheck.length < 1) {
```

Comment:
What sort of corner case are we trying to catch here? If it's unavoidable to fix elsewhere, can we 📝 this here?